admin_manual/configuration/server/occ_command.rst

```diff
@@ -843,16 +844,26 @@ Groups containing the ``search-pattern`` string are listed. Matching is
 not case-sensitive. If you do not provide a search-pattern then all groups 
 are listed.
 
-This example lists groups containing the string finance:: 
+Options:
+
+::
+  
+  --output[=OUTPUT]  Output format (plain, json or json_pretty, default is plain) [default: "plain"]
+
+This example lists groups containing the string finance
+
+:: 
  
  sudo -u www-data php occ group:list finance
   - All-Finance-Staff
   - Finance
   - Finance-Managers
 
-The output can be formatted in JSON with the output option ``json`` or ``json_pretty``::
+This example lists groups containing the string finance formatted in ``json_pretty``
```

Comment:
s/This example lists groups containing the string finance formatted in ``json_pretty``/This example lists groups containing the string "finance" formatted with ``json_pretty``/g